infrastructure/logging/build.gradle

```diff
@@ -6,6 +6,8 @@ dependencies {
     implementation 'org.apache.logging.log4j:log4j-core'
     compileOnly 'com.github.oshi:oshi-core'
 
+    testImplementation testFixtures(project(':ethereum:spec'))
```

Comment:
this is definitely an indication we're not in the right place... we're likely to be doing cyclic deps here if we're not very careful